.travis.yml

```diff
@@ -49,6 +49,14 @@ jobs:
       env:
         - BINARYBUILDER_RUNNER=docker
 
+    - stage: Documentation
+      julia: 1.0
+      os: linux
+      script:
+        - julia --project=docs -e 'using Pkg; Pkg.instantiate()'
```

Comment:
```suggestion
        - julia --project=docs -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate()'
```
?

Reply:
Nah, https://github.com/JuliaPackaging/BinaryBuilder.jl/pull/414/files#diff-17fe722019cf21306d74393750d3267aR6

Reply:
Ew, checking in a manifest 😦

Reply:
Checking in a manifest is good practice; it allows users to know a single point in package version space that at least worked on one person's machine, and allows CI to avoid breaking your package due to upstream changes.  Ideally, we would have CI that does both "use `Manifest`"-style and "install it like I just did `]add`"-style, but we're not quite there yet.  :)

Reply:
This is `docs/Manifest.toml` though, no reason not to lock doc-build dependencies.